tasks/push-snapshot/README.md

```diff
@@ -12,6 +12,11 @@ Tekton task to push snapshot images to an image registry using `cosign copy`.
 | addGitShaTag    | Also push a tag with the git sha for each image in the Snapshot                                 | Yes      | true                 |
 | addSourceShaTag | Also push a tag with the source sha for each image in the Snapshot                              | Yes      | true                 |
 | addTimestampTag | Also push a tag with the current timestamp for each image in the Snapshot                       | Yes      | false                |
+| tagPrefix       | The prefix used to build the tag in <prefix>-<timestamp> format                                 | Yes      | ""                   |
+
+## Changes since 0.11
+* The task now generates a tag in <prefix>-<timestamp> format when `tagPrefix` length is non zero
+* Adds `tagPrefix` parameter
```

Comment:
Nit: Switch the order of the lines - the new param is first and then based on that the task does something.

Reply:
done.